src/ADAL.PCL.CoreCLR/PlatformInformation.cs

```diff
@@ -61,5 +61,20 @@ public override string GetDeviceModel()
             // Since ADAL .NET may be used on servers, for security reasons, we do not emit device type.
             return null;
         }
+
+        public override string GetApplicationName()
+        {
+            return null;
+        }
+
+        public override string GetApplicationVersion()
+        {
+            return null;
+        }
+
+        public override string GetDeviceId()
```

Comment:
Its a abstract method in base class

---
In reply to: [73984749](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r73984749) [](ancestors = 73984749)